### PR TITLE
Chrome string translation preparations

### PR DESCRIPTION
**Pull request by Mailaender**
Split from https://github.com/OpenRA/OpenRA/pull/20594 where it helps homogenize label IDs between TD and all the others. It also localizes dynamic strings that can't be automatically extracted.

### CODE REVIEW

**Comment by abcdefg30** [comment]
This probably needs a rebase after #20804 was merged now.

**Comment by Mailaender** [comment]
Rebased.

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/ProductionTooltipLogic.cs [hunk 1]
```diff
@@ -19,8 +19,11 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class ProductionTooltipLogic : ChromeLogic
 	{
+		[TranslationReference("prequisites")]
+		const string Requires = "label-requires";
+
 		[ObjectCreator.UseCtor]
-		public ProductionTooltipLogic(Widget widget, TooltipContainerWidget tooltipContainer, Player player, Func<ProductionIcon> getTooltipIcon)
+		public ProductionTooltipLogic(ModData modData, Widget widget, TooltipContainerWidget tooltipContainer, Player player, Func<ProductionIcon> getTooltipIcon)
```
Comment (PunkPun) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/ProductionTooltipLogic.cs]: Unnecessary change

**Review by PunkPun — COMMENTED** [review]
Otherwise LGTM

File: mods/cnc/chrome/connection.yaml [hunk 2]
```diff
@@ -62,6 +62,7 @@ Container@CONNECTIONFAILED_PANEL:
 					Width: PARENT_RIGHT
 					Height: 25
 					Align: Center
+					Text: ServerError
```
Comment (abcdefg30) [inline mods/cnc/chrome/connection.yaml]: Since that is a panel title
```suggestion
					Text: Server Error
```
? (This should probably be adjusted in the common file as well.)

**Review by abcdefg30 — CHANGES_REQUESTED** [review]
All the chrome renamings need to be tested, this PR regresses the connection password dialogue for example:
image
On bleed it looks like this:
image

File: mods/cnc/chrome/connection.yaml [hunk 3]
```diff
@@ -45,7 +45,7 @@ Container@CONNECTIONFAILED_PANEL:
 			Font: BigBold
 			Contrast: true
 			Align: Center
-		Background@CONNECTION_BACKGROUND:
+		Background@CONNECTIONFAILED_PANEL:
```
Comment (PunkPun) [inline mods/cnc/chrome/connection.yaml]: ConnectionFailedLogic needs to be updated

File: mods/cnc/chrome/editor.yaml [hunk 4]
```diff
@@ -75,7 +75,7 @@ Container@NEW_MAP_BG:
 			Font: Bold
 			Key: return
 
-Container@SAVE_MAP_PANEL:
+Container@SAVE_MAP:
```
Comment (PunkPun) [inline mods/cnc/chrome/editor.yaml]: IngameMenuLogic needs to be updated

File: mods/common/chrome/editor.yaml [hunk 5]
```diff
@@ -72,7 +72,7 @@ Background@NEW_MAP_BG:
 			Font: Bold
 			Key: escape
 
-Background@SAVE_MAP_PANEL:
+Background@SAVE_MAP:
```
Comment (PunkPun) [inline mods/common/chrome/editor.yaml]: IngameMenuLogic needs to be updated

**Comment by Mailaender** [comment]
Removed the offending commit so we can proceed faster on the rest.

**Review by abcdefg30 — DISMISSED** [review]

File: mods/common/chrome/connection.yaml [hunk 6]
```diff
@@ -25,7 +25,7 @@ Background@CONNECTIONFAILED_PANEL:
 			Y: 76
 			Width: PARENT_RIGHT
 			Height: 25
-			Text: ServerError
+			Text: Server Error
```
Comment (pchote) [inline mods/common/chrome/connection.yaml]: Is this text even used? Better to remove it if not.

Comment (PunkPun) [reply mods/common/chrome/connection.yaml]: It's not used. It's always overwritten on widget creation.

The cnc chome doesn't even have this

**Review by PunkPun — APPROVED** [review]
LGTM

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/dbbac282e91c75a7dc04bd65dde78dafb2d9758a...c6da07386b5242f55ac1b58ded7fa32a45bd22a3)
